packages/suite/src/views/settings/index.tsx

```diff
@@ -343,14 +355,22 @@ const Settings = () => {
                                                         variant="tertiary"
                                                         icon="EXTERNAL_LINK"
                                                         alignIcon="right"
+                                                        isDev={isDev()}
                                                     >
                                                         {process.env.VERSION}
+                                                        {isDev() && '-dev'}
                                                     </VersionButton>
                                                 </VersionLink>
                                             </VersionTooltip>
                                         ),
                                     }}
                                 />
+                                {desktopUpdate.skip && (
```

Comment:
Has this been discussed with the product or is it just your idea?

If your idea, what about to get quick feedback from Product? 
![image](https://user-images.githubusercontent.com/3729633/134545800-d1f8908f-dc1d-44f3-b1ce-8971a52e821a.png)
#suite_ui channel maybe?